js/services/ajax.ts

```diff
@@ -272,7 +272,7 @@ export const uploadFiles = (
 };
 
 /**
- * POST to `/api/social/1/stream/streamId/file` to upload a file to a 3rd party social stream
+ * POST to `/api/run/1/social/stream/streamId/file` to upload a file to a 3rd party social stream
```

Comment:
I don't see any of these social endpoints in the api tool in production.

Reply:
AFAIK they are hidden from the Swagger definition - they definitely do exist in production though, otherwise social feeds wouldn't work.

Reply:
"AFAIK they are hidden from the Swagger definition" - so we should fix this.